javascript-data-model/main.js

```diff
@@ -0,0 +1,37 @@
+/* exported orderHistory */
+
+var orderHistory = [
+  {
+    orderNum: '114-3941689-8772232',
+    orderPlaced: '08/04/2020',
+    orderDelivered: '08/08/2020',
+    productName: ['JavaScript for impatient programmers'],
```

Comment:
Look at the data shown for the item in the example. Is a string description of the item name going to be enough information to build the entry in the output? I see an image for each entry, and some items have extra information such as an author. Something to consider!